packages/functions/package.json

```diff
@@ -27,10 +46,57 @@
     "@react-native-firebase/app": "23.5.0"
   },
   "devDependencies": {
-    "@react-native-firebase/private-tests-firebase-functions": "^0.0.1"
+    "@react-native-firebase/private-tests-firebase-functions": "^0.0.1",
+    "react-native": "^0.80.1",
+    "react-native-builder-bob": "^0.40.12",
+    "typescript": "^5.8.3"
```

Comment:
remove this and put path to TypeScript executable. See: https://github.com/invertase/react-native-firebase/pull/8651/files#diff-d1b85f8d1f6297c72a7aa703ba620c7eb120ea77be0538faed320a306eeab865R80